rules/Symfony73/NodeAnalyzer/LocalArrayMethodCallableMatcher.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Symfony73\NodeAnalyzer;
+
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\String_;
+use Webmozart\Assert\Assert;
+
+final class LocalArrayMethodCallableMatcher
+{
+    public function match(Expr $expr): ?string
+    {
+        if ($expr instanceof MethodCall) {
+            Assert::isInstanceOf($expr->name, Identifier::class);
+
+            return $expr->name->toString();
+        }
+
+        if ($expr instanceof Array_) {
+            $firstItem = $expr->items[0];
+            if (! $firstItem->value instanceof Variable) {
+                return null;
+            }
+
+            $methodName = $firstItem->value->name;
+            if (! is_string($methodName)) {
+                return null;
+            }
+
+            if ($methodName !== 'this') {
```

Comment:
Should be `$varObjectName` or something similar to avoid confusion of method name named "this"

Reply:
Could you resolve these? I'm already in another rule

Reply:
I will try

Reply:
I created PR : 

- https://github.com/rectorphp/rector-symfony/pull/767

for it.